pyFAST/fastfarm/examples/Ex3_FFarmCompleteSetup.py

```diff
@@ -20,100 +20,112 @@ def main():
+    # LESpath = '/full/path/to/the/LES/case'
+    LESpath = 'MannBox' # set as None if TurbSim-driven is desired
+    # LESpath = 'TurbSim' # set as None if TurbSim-driven is desired
+    
+    floating=1
     
     
     # -----------------------------------------------------------------------------
     # ----------- Template files
-    templatePath            = '/full/path/where/template/files/are'
+
+    templatePath            = 'path to template'
     
     # Put 'unused' to any input that is not applicable to your case
     # Files should be in templatePath
-    EDfilename              = 'ElastoDyn.T'
-    SEDfilename             = 'SimplifiedElastoDyn.T'
-    HDfilename              = 'HydroDyn.dat'
-    SrvDfilename            = 'ServoDyn.T'
-    ADfilename              = 'AeroDyn.dat'
-    ADskfilename            = 'AeroDisk.dat'
-    SubDfilename            = 'SubDyn.dat'
-    IWfilename              = 'InflowWind.dat'
-    BDfilepath              = 'unused'
-    bladefilename           = 'Blade.dat'
-    towerfilename           = 'Tower.dat'
-    turbfilename            = 'Model.T'
-    libdisconfilepath       = '/full/path/to/controller/libdiscon.so'
+    EDfilename              = 'Activefloat_ElastoDyn.T'
+    SEDfilename             = 'unused'
+    HDfilename              = 'Activefloat_HydroDyn.T'
+    MDfilename              = 'Activefloat_MoorDyn.T'
+    SrvDfilename            = 'Activefloat_ServoDyn.T'
+    ADfilename              = 'Activefloat_IEA-15-240-RWT_AeroDyn15.dat'
+    ADskfilename            = None
+    SubDfilename            = None
+    IWfilename              = 'Activefloat_IEA-15-240-RWT_InflowFile.dat'
+    BDfilepath              = None
+    bladefilename           = 'Activefloat_IEA-15-240-RWT_ElastoDyn_blade.dat'
+    towerfilename           = 'Activefloat_IEA-15-240-RWT_ElastoDyn_tower.dat'
+    turbfilename            = 'Activefloat.T'
+    libdisconfilepath       = 'DISCON.dll'
     controllerInputfilename = 'DISCON.IN'
-    coeffTablefilename      = 'CpCtCq.csv'
-    FFfilename              = 'Model_FFarm.fstf'
+    coeffTablefilename      = 'Cp_Ct_Cq.IEA15MW.txt'
+    FFfilename              = 'Activefloat_Farm.fstf'
     
     # TurbSim setups
-    turbsimLowfilepath      = './SampleFiles/template_Low_InflowXX_SeedY.inp'
-    turbsimHighfilepath     = './SampleFiles/template_HighT1_InflowXX_SeedY.inp'
+    turbsimLowfilepath      = '\SampleFiles\template_Low_InflowXX_SeedY.inp'
+    turbsimHighfilepath     = '\SampleFiles\template_HighT1_InflowXX_SeedY.inp'
     
     # SLURM scripts
-    slurm_TS_high           = './SampleFiles/runAllHighBox.sh'
-    slurm_TS_low            = './SampleFiles/runAllLowBox.sh'
-    slurm_FF_single         = './SampleFiles/runFASTFarm_cond0_case0_seed0.sh'
+    slurm_TS_high           = '\SampleFiles/runAllHighBox.sh'
+    slurm_TS_low            = '/runAllLowBox.sh'
+    slurm_FF_single         = '/runFASTFarm_cond0_case0_seed0.sh'

```

Comment:
I see how I left the `/` here for *nix systems and should have used `os.path.join` instead. But by changing this to the Windows convention, you break its use. This needs to be updated to a more generic approach (e.g. using `os`)

Reply:
yes I agree